src/seedu/addressbook/data/person/NameTest.java

```diff
@@ -0,0 +1,19 @@
+package seedu.addressbook.data.person;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class NameTest {
+
+	/**
+	 * Test isSimilar method in Name class
+	 */
+	
+	@Test
+	public void isSimilar_null_returnsFalse() {
+		Name testNull = new Name("John Doe");
```

Comment:
ensure to set eclipse to replace tabs with 4 spaces in the java style formatter.